float32[24] x
float32[24] vx
float32[24] ax

float32[24] y
float32[24] vy
float32[24] ay

float32[24] z
float32[24] vz
float32[24] az

float32[24] yaw
float32[24] vyaw
float32[24] ayaw

float32[24] duration
float32[24] wait_time
int32 points
